ts-defs/index.d.ts

```diff
@@ -756,6 +756,23 @@ interface TypeActionOptions extends ClickActionOptions {
     paste?: boolean;
 }
 
+interface DragToElementOptions extends MouseActionOptions {
+    /**
+     * Mouse pointer X coordinate that define a point where the dragToElement action is finished.
```

Comment:
Mouse pointer X coordinate that defines a point where the dragToElement action is finished.